Snowflake.Data.Tests/Snowflake.Data.Tests.csproj

```diff
@@ -30,15 +30,10 @@
     <Folder Include="Properties\" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net471'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'net471' Or '$(TargetFramework)' == 'net472'">
     <Reference Include="System.Net.Http" />
     <Reference Include="System.Web" />
     <Reference Include="System.Net.Http.WebRequest" />
-  </ItemGroup> 
-  
-  <ItemGroup Condition="'$(TargetFramework)' == 'net46'">

```

Comment:
are we sure this condition won't be met?

Reply:
Yes, we do not support the net 46 at all.